python/foglamp/services/core/server.py

```diff
@@ -582,8 +585,10 @@ def _check_readings_table(cls, loop):
 
     @classmethod
     def _start_core(cls, loop=None):
-        _logger.info("start core")
+        if cls.running_in_safe_mode:
+            _logger.info("Starting in safe mode")
 
+        _logger.info("start core")
```

Comment:
Let's  log  once. `_logger.info("Starting ...") and _logger.info("Starting in SAFE MODE...")` based on condition